Src/Sentry.Xamarin/Internals/NativeIntegration.droid.cs

```diff
@@ -51,5 +53,16 @@ private void Platform_ActivityStateChanged(object sender, ActivityStateChangedEv
                     ["state"] = e.State.ToString()
                 }, level: BreadcrumbLevel.Info);
         }
+        private void AndroidEnvironment_UnhandledExceptionRaiser(object sender, RaiseThrowableEventArgs e)
+        {
+            e.Exception.Data[Mechanism.HandledKey] = e.Handled;
```

Comment:
I also added this to Symbol Collector but still not sure when this kicks in and the AppDomain one doesn't.

I assume Java exceptions? Maybe next test is to create a .java file and `throw new RuntimeException()` there and see if it ends up here. I believe it will.